bedrock/firefox/templates/firefox/hello/index.html

```diff
@@ -218,6 +222,7 @@ <h3 class="heading-tertiary">{{ _('Get Firefox to get started') }}</h3>
     <div class="container">
       <div class="ctacopy-wrapper">
         <h2 class="dltry-copy heading-secondary" id="ctacopy-nonfx">{{ _('Get Firefox and start your first conversation') }}</h2>
+        <h2 class="dltry-copy heading-secondary" id="ctacopy-esrfx">{{ _('Sorry, Hello is not yet available in Firefox ESR.') }}</h2>
```

Comment:
Looks like Matej suggests `"You need a different version of Firefox to use Hello"` here: https://bugzilla.mozilla.org/show_bug.cgi?id=1164424#c3